config/mbed_lib.json

```diff
@@ -6,7 +6,10 @@
         "SPI_MISO": "D12",
         "SPI_CLK": "D13",
         "DEVICE_SPI": 1,
-        "FSFAT_SDCARD_INSTALLED": 1
+        "FSFAT_SDCARD_INSTALLED": 1,
+        "CRC_ENABLED": 1,
+        "CRC7": "SlowCRC",
+        "CRC16":"FastCRC"
```

Comment:
Out of curiousity, why is one fast and one slow?

Reply:
Both Fast and Slow support CRC7/CRC16. 

Using SlowCRC implementation for 7-bit as computation will be less (5bytes per command). SlowCRC does runtime computation.

FastCRC implementation generates CRC table during initialization instead of runtime computation, which makes it faster (but takes up RAM space). Using FastCRC for data makes sense as we need to compute for 512*blocks received for every transaction.

Reply:
That makes sense, thanks!

> but takes up RAM space

Can this be forced into ROM?

Reply:
Is there a chance we could profile an SD card read and see what percentage of runtime is spent calculating the CRC? I'm really curious how the CRC compares to the overhead of the SPI transaction.

Reply:
Yes we can check overhead in read as well as write data.

Reply:
And correct me if I'm wrong, but this is taking roughly 512 bytes of ROM?

Reply:
512 bytes for CRC16, 256 bytes for CRC7 if using FastCRC.